xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -375,73 +361,62 @@ void reactivate() {
       boolean isDeactivated() {
         return delayedDeletionTimer != null;
       }
-    }
-
-    class ChildHelper extends ForwardingLoadBalancerHelper {
 
-      private final OrcaReportingHelperWrapper orcaReportingHelperWrapper;
+      class ChildHelper extends ForwardingLoadBalancerHelper {
 
-      private SubchannelPicker currentChildPicker = XdsSubchannelPickers.BUFFER_PICKER;
-      private ConnectivityState currentChildState = CONNECTING;
+        private final OrcaReportingHelperWrapper orcaReportingHelperWrapper;
+        private SubchannelPicker currentChildPicker = XdsSubchannelPickers.BUFFER_PICKER;
+        private ConnectivityState currentChildState = CONNECTING;
 
-      ChildHelper(final Locality locality, final ClientLoadCounter counter,
-          OrcaOobUtil orcaOobUtil) {
-        checkNotNull(locality, "locality");
-        checkNotNull(counter, "counter");
-        checkNotNull(orcaOobUtil, "orcaOobUtil");
```

Comment:
nit: All diffs except these 5 lines in ChildHelper class are unrelated.

Reply:
Now `ChildHelper` is an inner class of `LocalityLbInfo`, these fields no longer need to be passed in. Can just directly access, without confusion of which instance they are referring to.

Reply:
I mean diffs other than these line.